### PR TITLE
fix(copy): Copy and design of Issue Owners page

### PR DESCRIPTION
**Pull request by robinrendle**
- Copy fixes to the modal to make it easier to read
- Copy fixes to settings page

### CODE REVIEW

File: static/app/components/modals/editOwnershipRulesModal.tsx [hunk 1]
```diff
@@ -14,7 +14,9 @@ type Props = ModalRenderProps & EditOwnershipRulesModalOptions;
 const EditOwnershipRulesModal = ({Body, Header, onSave, ...props}: Props) => {
   return (
     <Fragment>
-      <Header closeButton>{t('Edit Ownership Rules')}</Header>
+      <Header closeButton>
+        <h4>{t('Edit Ownership Rules')}</h4>
```
Comment (evanpurkhiser) [inline static/app/components/modals/editOwnershipRulesModal.tsx:18]: Should we change the styling on whatever the modal Header does

Comment (robinrendle) [reply static/app/components/modals/editOwnershipRulesModal.tsx:18]: yeah we definitely should but not in this PR, old bean

Comment (evanpurkhiser) [reply static/app/components/modals/editOwnershipRulesModal.tsx:18]: Do you want to actually change this to a h4 then, how many other form `Header`'s have h4s in them?

Comment (robinrendle) [reply static/app/components/modals/editOwnershipRulesModal.tsx:18]: Not in this PR

Comment (evanpurkhiser) [reply static/app/components/modals/editOwnershipRulesModal.tsx:18]: I mean are you sure you want it change this one here, when we might update all of them at once later?

I'm just arguing for consistency

**Review by scttcper — APPROVED** [review]

**Pull request merged**